test/actions/katello/product_test.rb

```diff
@@ -115,6 +115,10 @@ class UpdateTest < TestBase
                               product)
     end
 
+    it 'updates Candlepin when product content has changed' do
+      # is this expected behavior?
+    end
```

Comment:
Repository update is the only place that we update candlepin content:  https://github.com/Katello/katello/blob/master/app/lib/actions/katello/repository/update.rb#L14-L22

We should probably re-import the content as part of https://github.com/Katello/katello/blob/master/app/lib/actions/candlepin/product/content_update.rb

Reply:
Makes sense, and that's consistent with other code I've removed!